src/muxi/runtime/formation/server/server.py

```
@@ -522,8 +522,91 @@ async def validation_exception_handler(request, exc: RequestValidationError):
+                data={
+                    "component": "mcp_server",
+                    "formation_id": formation_id,
+                    "endpoint": "/mcp",
+                },
+                description="MCP server mounted at /mcp",
+            )
+            print(observability.InitEventFormatter.format_ok("MCP server", "mounted at /mcp"))
+
+        except ImportError:
+            pass
+        except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)